### DISCUSSION TITLE
Voice client in interactions

### DISCUSSION THREAD

**Discussion by IIDarkNessYT** [Rapptz/discord.py#8248] [Q&A]
Of course, I bombarded you with questions, but I'm making a music bot in slash commands using interactions. But I can't find in the documentation how to play audio through interactions. Can you provide an example or a link to the documentation?

**Comment by LostLuma** [comment]
Playing audio from interactions works the same way it does from regular commands, so you can use the existing documentation and the [basic voice example](https://github.com/Rapptz/discord.py/blob/master/examples/basic_voice.py) but replace the commands with slash commands.
